applications/luci-app-mwan3/htdocs/luci-static/resources/view/mwan3/mwan3.js

```diff
@@ -0,0 +1,95 @@
+function secondsToString(time) {
+	var seconds = parseInt(time, 10);
+
+	var hrs = Math.floor(seconds / 3600);
+	seconds -= hrs*3600;
+	var mnts = Math.floor(seconds / 60);
+	seconds -= mnts*60;
+	return String.format("%sh:%sm:%ss", hrs, mnts, seconds);
+}
+
+function renderMwan3Status(status) {
+
+	if (status.interfaces) {
```

Comment:
This is just a stylistic remark: Personally I prefer to use an "early return" pattern in functions which reduces the amount of indentation later on, like this:

```javascript
function renderMwan3Status(status) {
    if (!status.interface)
        return '<strong><%:No MWAN interfaces found%></strong>';

    var statusview = '';
    ...
}
```